pkg/ctl/update/cluster.go

```diff
@@ -15,12 +15,17 @@ import (
 	"github.com/weaveworks/eksctl/pkg/printers"
 )
 
-var updateClusterDryRun = true
+var (
+	updateClusterDryRun = true
+	updateClusterWait   = true
+)
 
 func updateClusterCmd(g *cmdutils.Grouping) *cobra.Command {
 	p := &api.ProviderConfig{}
 	cfg := api.NewClusterConfig()
 
+	// cfg.Metadata.Version = "next"
```

Comment:
What does `next` represent?

Reply:
Ah, nevermind I see the other comment below. I'm not super keen `next` that feels a little off. What is your thinking?

Reply:
I'm not really sure what else would be good for saying next version. I don't think we'd be able to have no version given have a default.

Reply:
This is documented in #596. The idea is to have `next` (default), `latest` and _semver_. But I'm not entirely sure I'll manage to implement more then one option in this PR, so I left it commented out as as there isn't much use to a flag that only supports a default value.

Reply:
Another reasons for leaving this until later is that currently there is only one upgrade path to test _1.10 - 1.11_, which implies that `--version=next` as well as  `--version=latest` as well as  `--version=1.11` do the same thing.